cmd/main.go

```diff
@@ -150,6 +143,9 @@ func main() {
 		banpInformer,
 		cfg,
 	)
+	if err != nil {
+		panic(err.Error())
+	}
```

Comment:
Why panic when it was `klog.Fatalf` before?
Semantically, this doesn't seem panic-worthy

Reply:
yeah it has to exit with error because the controller is not able to start